app/frontend/static/views/partials/search.html

```diff
@@ -53,9 +76,9 @@
   <br/>
   <div ng-repeat="topic in results.docs" class="row">
     <div class="col-md-9">
-      <h4><a href="{{topic.listing_url}}">{{topic.title}}</a></h4>
-      <p>{{topic.description|truncate:500}}<span ng-if="topic.description.length > 500"><a href="{{topic.listing_url}}">MORE >>></a></span></p>
-      <button class="btn btn-default btn-sm pull-right" ng-click="saveOpportunity(topic.id)" type="button">Save for later</button>
+      <h4><a href="topic/{{topic.id}}">{{topic.title}}</a></h4>
```

Comment:
Actually, it can be whatever. That url is handled entirely by the front end app - only requests to the `api` endpoint make requests to the python app.

I might go ahead and change it just for the sake of clarity, though. :)